etc/kayobe/ansible/requirements.yml

```diff
@@ -1,7 +1,7 @@
 ---
 collections:
   - name: stackhpc.cephadm
-    version: 1.10.0
+    version: 1.12.1
```

Comment:
I'm still concerned about this. It introduces the `cephadm_host_labels` bug which is going to affect all deployments and is annoying to diagnose. I'd much rather we had a fix in the collection.

Reply:
Probably the way to fix it is using `| default([])` when referencing the variable in the collection.